base/scratch_buffer.jl

```diff
@@ -0,0 +1,56 @@
+# NOTE: ryu/Ryu.jl: neededdigits(::Type{Float64}) = 309 + 17
+const SCRATCH_BUFFER_INLINE_LENGTH = 309 + 17
+struct ScratchBufferInline
+    data::NTuple{SCRATCH_BUFFER_INLINE_LENGTH,UInt8}
+
+    ScratchBufferInline(::UndefInitializer) = new()
```

Comment:
Just make it initialize with zero instead or alt make this a zero arg method? `UndefInitializer` is not really used for this, note the docs:

```
UndefInitializer

  Singleton type used in array initialization, indicating the
  array-constructor-caller would like an uninitialized array.
```

Reply:
fixed